www/app/Jobs/ProcessConversationStream.php

```
@@ -553,6 +566,22 @@ private function streamWithToolLoop(
                     break;
 
                 case StreamEvent::ERROR:
+                    if (
+                        $conversation->provider_type === 'claude_code'
```

PHPStan (2.1.46)

[error] Access to an undefined property App\Models\Conversation::$provider_type.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)